pvlib/tracking.py

```diff
@@ -43,16 +43,26 @@ class SingleAxisTracker(PVSystem):
         between the tracking axes has a gcr of 2/6=0.333. If gcr is not
         provided, a gcr of 2/7 is default. gcr must be <=1.
 
+    side_slope : float, default 0.0
+        The slope of the "system plane" perpendicular to the tracker axes. The
```

Comment:
```suggestion
        The slope of the line in the "system plane" perpendicular to the tracker azimuth. The
```

A plane doesn't have a slope. A plane perpendicular to the tracker axes is equivalent to saying that the plane's normal is parallel to the axes, which I don't think is what is meant here.

Reply:
Oops I think I merged this change too quickly, but perhaps it should be reworded, because it should be something like "the slope relative to horizontal of the line formed by the intersection between the "system plane" _and_ in a plane perpendicular to the tracker axis." (That's a mouthful.) In order to use `singleaxis()` everything needs to be in the reference frame of the trackers (tx, ty, tz), and this is explained at the end of section 5:
![image](https://user-images.githubusercontent.com/1385621/92178694-f321b900-edf7-11ea-84dc-98e2f05568a8.png)